agent/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Instrumentation Agent for Java
+
+[![Build Status][travis-image]][travis-url] [![Maven Central][maven-image]][maven-url]
+
+The *Google Instrumentation Agent for Java* collects and sends latency data
+about your Java process to Stackdriver Trace for analysis and visualization
```

Comment:
nit.. need to wash some of this stuff for the opencensus branding. Ex preferring wording like "opencensus backends such as StackDriver" vs being the only target.